src/appleseed.studio/python/module.cpp

```diff
@@ -135,4 +136,6 @@ BOOST_PYTHON_MODULE(_appleseedstudio)
 
     bpy::def("main_window", main_window_as_pylong);
     bpy::def("create_dock_widget", create_dock_widget, bpy::args("dock_name"));
+
+    boost::python::def("get_studio_dir", get_executable_directory);
```

Comment:
I think that the name is of this function is not quite correct. For me the studio directory would be the parent of the bin directory, the root of the appleseed install if you install appleseed using the released versions.

We could rename the function to make it clear that it returns the bin directory or we can make it return the root directory of the appleseed install. I think I prefer the last option myself.